cli/src/index.py

```diff
@@ -80,12 +64,6 @@
 if "API_KEY" not in environ or len(environ["API_KEY"]) == 0:
     CREDENTIALS = False
 
-if "WEBSITE_USERNAME" not in environ or len(environ["WEBSITE_USERNAME"]) == 0:
-    ADMIN = False
-
-if "WEBSITE_PASSWORD" not in environ or len(environ["WEBSITE_PASSWORD"]) == 0:
-    ADMIN = False
-
```

Comment:
So there is no way to set ADMIN to False, so ADMIN is always True, so the variable ADMIN is useless and the "if statement" is useless too.

=> Remove ADMIN, remove "if ADMIN" below